### ISSUE TITLE
best practise resource gone

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#1722]
**best practise resource gone**

we have links to GBIF resources describing best practices on how to publish data that vanished in the past year. Should these URLs not be a little bit more stable?

`https://github.com/Sp2000/colplus/blob/master/docs/DWCA.md#checklist-darwin-core-archives`

-----

System: Chrome 71.0.3578 / Mac OS X 10.14.2
User: [See in registry](https://www.gbif.org/api/feedback/user/28fae39fe6b6b7351a0da8c423bde085:a3753546dcf3ccb30b1e4fa6f09f4431a99c105d4423d9b9e58ad5ef9c95e9f8ec50163b8f50f8d43f9ab27a16b18e630f51ec578c94485c82d455c70c78fd9b)
Referer: https://www.gbif.org/resources/2514
Window size: width 1583 - height 822
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2019-01-11T08:13:18.129Z',mode:absolute,to:'2019-01-11T08:19:18.129Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2019-01-11T08:13:18.129Z',mode:absolute,to:'2019-01-11T08:19:18.129Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by kcopas** [comment]
I think it's been more than a year, @mdoering ! 

> Sadly they contradict each other in some areas and there is not a single, complete and up to date document available. This is maintained with the code and if in doubt should contain the most up to date answer.

Based on Kyle Braak's analysis, we removed several old and woefully out-of-date documents. We are on the verge of having some expert contractor prepare five different titles that will begin to clear the backlog of updating these with replacements (as digital-first documents maintained on GitHub). We're also establishing an editorial panel that will help us prioritize subsequent documentation needs (a second tranche is planned for later in 2019).

I don't know whether you're saying that all five of them listed here are missing. I can provide the specific rationale for removing any that are missing. I'd also be happy to provide you with copies to post locally in the interim. But I'm not inclined to restore them given the major issues they have.

**Comment by mdoering** [comment]
Yeah, I recall now. I am still missing a stable place to refer to what Darwin Core Checklist Archives are. TDWG only has the core parts (DwC terms, DwC archive structure) which are not enough. And GBIF lacks a comprehensive page explaining what that format exactly is with all its common extensions.
